src/server_manager/install_scripts/install_server.sh

```diff
@@ -41,6 +41,11 @@
 
 set -euo pipefail
 
+if [[ $(uname -m 2> /dev/null) != x86_64 ]]; then
```

Comment:
Please move this to the beginning of `install_shadowbox()`. Code execution on the global scope is hard to track.